### ISSUE TITLE
SIL Get File

### ISSUE THREAD

**Issue by rydougherty** [ombegov/ITDB-schema#286]
Name: Ryan Dougherty
Organization: Booz Allen, supporting eCPIC PMO
Urgency: High- dependencies on our end
Description of Issue: The Get File for the Systems Inventory List mandates that we submit UII along with System Name and System ID. Could you allow the Get File to submit just with the UII to create alignment with the structure of all other Get Files and OMBs communicated approach of Systems under UIIs? 

The current process of requiring a System ID will not work as is anyway. Unless you are using an Agency ID, the System ID is assigned after the first submission, so the first time you run the Get File for an investment, you will either encounter an Schema Error for dropping the node or a Server Error for sending a node with a blank value.

**Comment by courtneycorwin** [comment]
We might need to discuss this further.  While I understand Ryan's issue and agree that it needs to be addressed, we've discussed several times that an SIL submission is one investment and one system, so we need a way to determine if that combination exists, not just whether the UII does.  

If the submission is changed to be inline with how the guidance describes it (i.e. the report is submitted by investment and will contain several systems similar to the Contracts Report or how we report SOCs for the IT Security & Compliance SIR), I think it might better meet the needs of agencies that use the OMB assigned ID as well as those that use the agencyID.

In short, I'm fine (and very supportive) of this report being by investment and having the add/update operation be on the system, but then we shouldn't need to include the UII with every system node.

**Comment by austin-moffa** [comment]
Courtney's read on this matches my understanding as well. Since a single UII can have multiple inventories associated with it, we require another key in combination to pull a specific report. The upcoming 8A sprint contains a fix to prevent the system error when submitting with empty id elements.

**Comment by jeremiah-tcg** [comment]
Can you confirm if this still an issue?

**Comment by elliotyoon202** [comment]
Hello,
Please let us know if this issue can be closed or if you're still experiencing this issue.

-Elliot
